api/user-story/models/user-story.js

```diff
@@ -22,35 +22,38 @@ module.exports = {
             link: `story/${result.id}`
           })
         }
-      }
-
-      catch(e) {}
+      } catch (e) {}
     },
-    async beforeUpdate(params,data) {
+    async beforeUpdate(params, data) {
       try {
-           const story=await strapi.query('user-story').findOne({id: params.id})
-           const userId= data.followers[data.followers.length-1]
-           if(story?.followers?.length < data?.followers?.length && story.author.id !== userId) {
-           const user= await strapi.query('user', 'users-permissions').findOne({id: userId})
-           await strapi.services['user-story-notification'].create({
+        const story = await strapi.query('user-story').findOne({
+          id: params.id
+        })
+        const userId = data.followers[data.followers.length - 1]
+        if (story?.followers?.length < data?.followers?.length && story.author.id !== userId) {
+          const user = await strapi.query('user', 'users-permissions').findOne({
+            id: userId
+          })
+          await strapi.services['user-story-notification'].create({
             message: `${user.username} voted up your story`,
             users: [story.author],
             date: new Date(),
             link: `story/${story.id}`
-           })
-          }
-      }
-      catch(e) {}
+          })
+        }
+      } catch (e) {}
     },
     async beforeCreate(data) {
       if (!data.user_story_status) {
         try {
-          const status = await strapi.query('user-story-status').findOne({ Status: 'Under consideration' })
+          const status = await strapi.query('user-story-status').findOne({
+            Status: 'Under consideration'
+          })
           data.user_story_status = status.id
         } catch (e) {
           console.log('Unable to fetch "Under consideration" status')
         }
       }
     }
   }
-};
+};
```

Comment:
Please add an empty line at the end of the file

Reply:
Done.